console_conf/ui/views/welcome.py

```diff
@@ -51,6 +51,13 @@ def confirm(self, result):
 
 class ChooserWelcomeView(WelcomeView):
     excerpt = (
-        "System recovery triggered. Proceed to select one of available "
-        "systems and execute a recovery action."
+        "System mode selection triggered. Proceed to select one of available "
+        "systems and actions."
     )
+
+    def __init__(self, controller):
+        current = controller.model.current
+        if current is not None:
+            self.title = current.model.display_name
+
+        super().__init__(controller)
```

Comment:
Could this init happen before we assign to `self.title`?

Reply:
Fixed this and the other one.